NitroxClient/Debuggers/Drawer/Nitrox/NitroxEntityDrawer.cs

```diff
@@ -47,5 +49,21 @@ private static void DrawNitroxId(NitroxId nitroxId)
             NitroxGUILayout.Separator();
             GUILayout.TextField(nitroxId == null ? "ID IS NULL!!!" : nitroxId.ToString());
         }
+
+        GUILayout.Space(8);
+
+        using (new GUILayout.HorizontalScope())
+        {
+            GUILayout.Label("Simulating state", GUILayout.Width(LABEL_WIDTH));
```

Comment:
I know we haven't got it on these so far but do we want to have the multi language support on them?

Reply:
I would definitely say no. These are debug tools only for us only.

Reply:
I was thinking that as well, just thought I would see people's appetite. 

Also being a dev tool add and removing template strings would be a pain